src/Microsoft.SqlTools.ServiceLayer/EditData/SmoEditMetadataFactory.cs

```diff
@@ -99,6 +99,12 @@ public EditTableMetadata GetObjectMetadata(DbConnection connection, string[] obj
             {
                 Column smoColumn = smoResult.Columns[i];
 
+                // Don't return columns that are dropped
+                if (smoColumn.IsDroppedLedgerColumn)
```

Comment:
@shueybubbles - This shouldn't have any perf implications just querying the value on the column, right? (especially since we already refresh it above)

@nofield Did you test this with a older sql version that doesn't have ledger stuff? I thought the default for unsupported properties was to throw but I may be misremembering that.

Reply:
`IsDroppedLedgerColumn` is hardcoded to false on old versions
See https://github.com/microsoft/sqlmanagementobjects/blob/73cd2cb929409cb9ddc2c7b416c5d7c6ef621d5f/src/Microsoft/SqlServer/Management/SqlEnum/xml/column.xml#L247

Reply:
In SMO the property is defined for all versions of SQL, just defaulted to 0 if it's an unsupported sql version. https://msdata.visualstudio.com/SQLToolsAndLibraries/_git/SqlManagementObjects?path=/src/Microsoft/SqlServer/Management/SqlEnum/xml/column.xml&version=GBmaster&line=248&lineEnd=249&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents